test/Integration/SqlTriggerBindingIntegrationTests.cs

```diff
@@ -97,13 +97,51 @@ public async Task MultiOperationTriggerTest()
             changes.Clear();
         }
 
+        /// <summary>
+        /// Ensures correct functionality with user functions running across multiple functions host processes.
+        /// </summary>
+        [Fact]
+        public async Task MultiHostTriggerTest()
+        {
+            this.EnableChangeTrackingForTable("Products");
+
+            var changes = new List<SqlChange<Product>>();
+            DataReceivedEventHandler[] changeHandlers = new[] { this.GetProductChangeHandler(changes, "SQL Changes: ") };
+
+            // Prepare three function host processes
+            this.StartFunctionHost(nameof(ProductsTrigger), Common.SupportedLanguages.CSharp, useTestFolder: false, changeHandlers);
+            this.StartFunctionHost(nameof(ProductsTrigger), Common.SupportedLanguages.CSharp, useTestFolder: false, changeHandlers);
+            this.StartFunctionHost(nameof(ProductsTrigger), Common.SupportedLanguages.CSharp, useTestFolder: false, changeHandlers);
+
+            // Considering the polling interval of 5 seconds and batch-size of 10, it should take around 15 seconds to
```

Comment:
If we update the default batch-size and polling interval, I would like to have those reflected in all of the test cases, as the tests are in a way testing that those defaults are being honored correctly. For example, this test case also ensures that all changes are consumed 3x faster since as there are 3 functions hosts doing the work. Though not the main reason, setting the exact condition for exiting-out the polling will become tricky for 'multiple functions, single host' case (the other PR) where we will need to check that every function has completed consuming the change at its end.